x-pack/test/functional/page_objects/infra_hosts_view.ts

```diff
@@ -28,6 +28,10 @@ export function InfraHostsViewProvider({ getService }: FtrProviderContext) {
       return await testSubjects.click('euiFlyoutCloseButton');
     },
 
+    async clickProcessesFlyoutTab() {
+      return await testSubjects.click('hostsView-flyout-tabs-processes');
+    },
```

Comment:
**nit**: the `return await` is redundant, return directly the promise:
```js
clickProcessesFlyoutTab() {
  return testSubjects.click('hostsView-flyout-tabs-processes');
},
```

Reply:
Ah, sorry about all of those (I was copying/pasting stuff around from different places and forgot to remove the wait after 🤦‍♀️ )